floris/heterogeneous_map.py

```diff
@@ -195,12 +235,49 @@ def get_heterogeneous_inflow_config(
                 self.speed_multipliers, len(wind_directions), axis=0
             )
 
-        # Return heterogeneous_inflow_config
-        return {
-            "x": self.x,
-            "y": self.y,
-            "speed_multipliers": speed_multipliers_by_findex,
-        }
+        # Return heterogeneous_inflow_config with only x and y is z is not defined
+        if self.z is None:
```

Comment:
I think here, we could just have a single return statement that always returns the `"z": self.z` field, even if it is None. Would be more consistent and a simpler output.

Reply:
I think it would work, but I think most of the code I've looked at expects that a 2d heterogeneous_inflow_config has no 'z' field